app/src/main/res/layout/content_main.xml

```diff
@@ -30,31 +31,28 @@
             android:paddingBottom="10dp">
 
             <Button
-                android:text="<"
```

Comment:
fixed XML issue, but didn't change layout. Text of button is set programmatically now.